public/app/features/search/tempI18nPhrases.ts

```diff
@@ -0,0 +1,10 @@
+// Temporary place to collect phrases we reuse between new and old browse/search
+// TODO: remove this when new Browse Dashboards UI is no longer feature flagged
+
+import { t } from 'app/core/internationalization';
+
+export function getSearchPlaceholder(includePanels = false) {
+  return includePanels
+    ? t('search.search-input.include-panels-placeholder', 'Search for dashboards and panels')
+    : t('search.search-input.placeholder', 'Search for dashboards');
```

Comment:
old search has this problem as well, but this searches for dashboards **and** folders (and panels if that feature flag is enabled). would be good to update this to not just reference dashboards 👍